client/X11/xf_window.h

```diff
@@ -109,6 +109,10 @@ struct xf_app_window
 	INT32 clientOffsetY;
 	UINT32 clientAreaWidth;
 	UINT32 clientAreaHeight;
+    UINT32 windowLeftResizeMargin;
```

Comment:
Might be nice to fix indentation here if you are at it ;)

Reply:
Sure, no problem. (My editor put in spaces instead of a tab.)